assets/src/design-system/components/contextMenu/animatedContextMenu.js

```diff
@@ -238,6 +241,21 @@ function AnimationContainer({ children, isOpen, ...props }) {
     setAlign(CORNER_DIRECTIONS[`${alignVertical}_${alignHorizontal}`]);
   }, [isOpen]);
 
+  useEffect(() => {
+    // some styles depend on the animation being finished. Set a timeout to set this variable
+    // once the animation has finished.
+    let timeoutId;
+    if (isOpen) {
+      setAnimationFinished(false);
+      timeoutId = setTimeout(
+        () => setAnimationFinished(true),
+        animationTimeSeconds * 1000
+      );
+    }
+
+    return () => clearTimeout(timeoutId);
+  }, [isOpen]);
```

Comment:
Removes the `overflow: hidden;` style from the container so that we can see the entire focus border

|Before|After|
|--|--|
|![image](https://user-images.githubusercontent.com/22185279/113457841-16343b80-93ce-11eb-83ab-62423e3ef127.png)|![image](https://user-images.githubusercontent.com/22185279/113457851-20563a00-93ce-11eb-801f-9fe85ecad746.png)|